### ISSUE TITLE
Add marker input to this sim

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab#249]
Related to https://github.com/phetsims/a11y-research/issues/153, we want to prototype in this sim with marker input and creating tangible controls with paper and computer vision.

**Comment by zepumph** [comment]
I created a simple slider by making an input group (anchor 21) and and a slider (marker 15). Here is the generated `inputs.txt`

<details>

```json
{"groups":[{"name":"mass1Position","anchorID":21,"detectWindow":250,"inputs":[{"name":"slider","type":"SLIDER","actorID":15,"detectWindow":250,"relativePosition":{"distance":37.75857347255701,"angle":-2.8807969315904076},"endPosition":{"distance":157.3037159237813,"angle":-2.111760785125986}}],"markerSize":20}]}
```
</details>


Then I was able to create a private type called `SliderMarker` that could handle that input group and set an AXON/Property based on the slider value. The output of the above commit is that three Properties are controlled by my one tangible slider, but that was only as a proof of concept. 

From here I think that the design side should take a stab at a few ideas that we actually want to see, because my feelings are that most of them will be pretty straight forward to code up so long as they are working with mechamarkers.
